src/Model/Item.php

```diff
@@ -4265,4 +4265,22 @@ public static function getQuoteUriId(string $body, int $uid = 0): int
 		Logger::warning('Post does not exist although it was supposed to had been fetched.', ['id' => $id, 'url' => $url, 'uid' => $uid]);
 		return 0;
 	}
+
+	public static function countInbound(string $network)
+	{
+		$packets = DI::keyValue()->get('stats_packets_inbound_' . $network) ?? 0;
+		if ($packets >= 2147483647) {
```

Comment:
Please use the `PHP_INT_MAX` constant to accommodate both 32-bit and 64-bit systems.
```suggestion
		if ($packets >= PHP_INT_MAX) {
```

Reply:
Done